src/metabase/util/schema.clj

```diff
@@ -106,6 +106,10 @@
 ;;; |                                                 USEFUL SCHEMAS                                                 |
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
+(def AllowedBlankString
+  "Schema for a string."
+  (s/named (s/cond-pre s/Str) (tru "String")))
```

Comment:
You definitely don't need to, and shouldn't define this. If you want a string that can be blank, you can just use `s/Str` directly. 

At any rate, [`cond-pre`](https://github.com/plumatic/schema#other-schema-types) is only for creating a conditional schemas from one or more schemas -- i.e. to allow something that satisfies any of several different schemas (e.g. a `String` OR an `Integer`).